src/service/action-impl.js

```diff
@@ -134,6 +135,8 @@ export class ActionInvocation {
     this.args = args;
     /** @const {?Element} */
     this.source = source;
+    /** @const {?Element} */
+    this.caller = caller;
```

Comment:
Would you mind document these params with a short description? At least for `target`, `source` and `caller`.

Reply:
Done.